### PR TITLE
Rat Polls

### PR DESCRIPTION
**Pull request by Gala-G**

### CODE REVIEW

File: governance/polls/Ratification Poll - Adding Lending Oversight Core Unit (LOVE-001) - June 13, 2022.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp72-lending-oversight-core-unit-mkr-budget-love-001/15104)
+
+### MIP41c4-SP36
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*I, @luca_pro, have been an active member of MakerDAO's community for a while, helping defining Maker's role along the crypto lending stack (also through the publication of the [Real-World Sandbox Report](https://forum.makerdao.com/t/real-world-sandbox-final-deliverable-for-consultation-round/11915?u=luca_pro)) as well as collaborating with several Core Units. I have an extensive 15+ years experience at the highest level of banking and fintech, and am a DeFi researcher through my Substack publication [Dirt Roads](https://dirtroads.substack.com/). I am a firm believer of MakerDAO's centrality within the DeFi ecosystem and intend to do everything I can to support this vision and the evolution of the DAO as Facilitator of LOVE-001.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/626570f9e3e20708f6223a463b6e839259e91c76)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Adding Lending Oversight Core Unit (LOVE-001) - June 13, 2022.md:69]: Use this URL: https://github.com/makerdao/mips/blob/626570f9e3e20708f6223a463b6e839259e91c76/MIP41/MIP41c4-Subproposals/MIP41c4-SP36.md

File: governance/polls/Ratification Poll - Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022
+summary: The Makershire Hathaway SPF will be a 10M pool managed by the SF CU under the supervision of Risk CU and prospective LOVE CU to experiment with various yield generation strategies and support DAI liquidity/utility.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022.md:3]: "The Makershire Hathaway SPF will be a 10M pool managed by the Strategic Finance CU (SF-001) under the supervision of the Risk CU (RISK-001) and the prospective Lending Oversight CU (LOVE-001) to experiment with various yield generation strategies and support DAI liquidity/utility"

File: governance/polls/Ratification Poll - Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP55c3-SP4 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Maker Protocol holds over $5.4B in stablecoins earning zero yield. There are multiple efforts underway (short-term bonds 1, IVs, RWA, Multi-chain) to invest the majority of stablecoin exposure into growth opportunities. Our aim is to begin experimenting with generating yield on balance sheet assets and support DAI liquidity/utility, creating a new revenue stream for the protocol. As DeFi is evolving quickly, we should continue researching, investigating, and experimenting with DeFi-native yield-generation strategies.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/24779f52630df945595594759e4cb4b757f4c177)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022.md:34]: Use this URL: https://github.com/makerdao/mips/blob/24779f52630df945595594759e4cb4b757f4c177/MIP55/MIP55c3-Subproposals/MIP55c3-SP4.md

File: governance/polls/Ratification Poll - Onboard Task Force - Growth Task Force (MIP75c3-SP1) - June 13, 2022.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP75c3-SP1 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*It is our view that we are in a nascent stage of crypto, and controlling as much future market share as possible is critical for the survival of this project. Our focus is on practical solutions to achieve the objectives listed below as quickly as possible. Perfection is the enemy of progress. We aim to find solutions that work first and backfill decentralization / resilience later.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/24779f52630df945595594759e4cb4b757f4c177)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Onboard Task Force - Growth Task Force (MIP75c3-SP1) - June 13, 2022.md:34]: Use this URL: https://github.com/makerdao/mips/blob/24779f52630df945595594759e4cb4b757f4c177/MIP75/MIP75c3-Subproposals/MIP75c3-SP1.md

File: governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Onboard/Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022
+summary: This is a budget amendment application for Keep3r Network: a decentralized and permissionless keeper network.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md:3]: Not sure if these fields can contain a colon. @prose11 ?

File: governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP63c4-SP4 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*During the 1 month period our network was active on Maker’s Keeper Network, we had an expenditure of 1.48 ETH (4800 USD at the time of writing), we expect this cost to be within that range (4,000 - 5,000 USD per month) and thus why we’ve decided to suggest a 4,000 Min Buffer limit. The suggested 20,000 Max Buffer limit is taken into account due to potential gas spikes and the likelihood of more jobs coming to the network within the next few months.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/24779f52630df945595594759e4cb4b757f4c177)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md:34]: Use this URL: https://github.com/makerdao/mips/blob/24779f52630df945595594759e4cb4b757f4c177/MIP63/MIP63c4-Subproposals/MIP63c4-SP4.md

File: governance/polls/Ratification Poll - Task Forces (MIP75) - June 13, 2022.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP75 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP75 proposes the formalization of Task Forces. These groups are designed to directly interface with the delegates to coordinate and agree upon terms in advance to provide more autonomy within the Core Units. Task Forces can be formed by anyone, but the ratification process needs to be put to a vote due to discretionary fund requests being attached. Task Forces are free to add or remove members by their own consensus rules.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/626570f9e3e20708f6223a463b6e839259e91c76)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Task Forces (MIP75) - June 13, 2022.md:34]: Use this URL: https://github.com/makerdao/mips/blob/626570f9e3e20708f6223a463b6e839259e91c76/MIP75/MIP75.md

**Review by blimpa — CHANGES_REQUESTED** [review]
URL changes for canonical proposal versions

File: governance/polls/Ratification Poll - Adding Lending Oversight Core Unit (LOVE-001) - June 13, 2022.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Lending Oversight Core Unit (LOVE-011) being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP33
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Adding Lending Oversight Core Unit (LOVE-001) - June 13, 2022.md:29]: Maybe also add title to these

File: governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md (hunk 5 above)
Comment (Gala-G) [reply governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md:3]: this is why I had to change the file title...

File: governance/polls/Ratification Poll - Adding Lending Oversight Core Unit (LOVE-001) - June 13, 2022.md (hunk 8 above)
Comment (Gala-G) [reply governance/polls/Ratification Poll - Adding Lending Oversight Core Unit (LOVE-001) - June 13, 2022.md]: Done

File: governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md (hunk 5 above)
Comment (patrick-j-govalpha) [reply governance/polls/Ratification Poll - Onboard | Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md:3]: For reference, the "|" was also breaking it, I've changed it to "Onboard-Amend Keep3r Network (MIP63c4-SP4) - June 13, 2022.md"

**Pull request merged**